draft-ietf-scitt-architecture.md

```diff
@@ -471,9 +474,9 @@ For a software supply chain, payloads describing the software artifacts may incl
 Once all the Envelope headers are set, an Issuer MUST use a standard COSE implementation to produce an appropriately serialized Signed Statement.
 The SCITT tag `COSE_Sign1_Tagged` is outside the scope of COSE, and used to indicate that a signed object is a Signed Statement.
 
-Issuers may produce Signed Statements about different Artifacts under the same Identity.
+Issuers MAY produce Signed Statements about different Artifacts under the same Identity.
```

Comment:
This is a throwaway, consider just removing this normative statement.

Reply:
Keeping this line puts context to how multiple Signed Signed Statements can be used. 
Changing to a colocial "can" to keep the description, without making it vague normative language. 
```suggestion
Issuers can produce Signed Statements about different Artifacts under the same Identity.
```